runtime/runtime-standalone/src/lib.rs

```diff
@@ -0,0 +1,394 @@
+use near_pool::{types::PoolIterator, TransactionPool};
+use near_primitives::{
+    account::Account,
+    errors::RuntimeError,
+    hash::CryptoHash,
+    receipt::Receipt,
+    state_record::StateRecord,
+    transaction::{ExecutionOutcome, ExecutionStatus, SignedTransaction},
+    types::AccountId,
+    types::{Balance, BlockHeight, EpochHeight, Gas, StateChangeCause},
+};
+use near_runtime_configs::RuntimeConfig;
+use near_store::{get_account, set_account, Store, Trie, TrieUpdate};
+use node_runtime::{state_viewer::TrieViewer, ApplyState, Runtime};
+
+use std::collections::HashMap;
+use std::sync::Arc;
+
+#[derive(Debug, Default)]
+pub struct GenesisConfig {
+    pub genesis_time: u64,
+    pub gas_price: Balance,
+    pub gas_limit: Gas,
+    pub runtime_config: RuntimeConfig,
+    pub state_records: Vec<StateRecord>,
+}
+
+#[derive(Debug, Default, Clone)]
+pub struct Block {
+    prev_block: Option<Box<Block>>,
+    state_root: CryptoHash,
+    gas_burnt: Gas,
+    pub epoch_height: EpochHeight,
+    pub block_height: BlockHeight,
+    pub block_timestamp: u64,
+    pub gas_price: Balance,
+    pub gas_limit: Gas,
+}
+
+impl Block {
+    pub fn genesis(genesis_config: &GenesisConfig) -> Self {
+        Self {
+            prev_block: None,
+            state_root: CryptoHash::default(),
+            block_height: 1,
```

Comment:
genesis block height is 0

Reply:
To be precise, `genesis_config.genesis_height`. Let's use that instead of `1`

Reply:
fixed, thanks!